docs/product-getting-started-guide/src/main/asciidoc/business-rules-create-proc.adoc

```diff
@@ -0,0 +1,73 @@
+[id='guided-rule-licence-suspension-create-proc']
+= Creating the Driver License Suspension rule
+
+The following chapter instructs you how to create a driver license suspension rule in {PRODUCT} using the Guided Rule wizard.
+
+[float]
+== Prerequisite
+
+Created both the Violation and Driver data objects.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Guided Rule*, then enter:
+
+ * *Guided Rule*: `DriverLicenseSuspensionRule`
+ * *Package*: `org.jboss.example.traffic_violations`
++
+
+. Click *+Ok* to open the *Guided Rule designer*.
++
+
+.Create new Guided Rule window
+image::guided-rule.png[]
+
+== Setting the Suspension rule conditions
+
+. Click image:6176.png[] next to the *WHEN* label to open the *Add a condition to the rule* window. . . Select *Driver...* and click *Ok*.
```

Comment:
Search all instances of ellipsis (...) and address. Per IBM SG, don't include an ending ellipsis even if it's in the UI element. The only exception I've found so far is "(show options...)" because technically the ellipsis is not the last trailing thing, has parentheses. Just sayin'.

Reply:
done and noted for future ref :)